src/Jade/Engine/PugJsEngine.php

```diff
@@ -54,7 +54,15 @@ protected function getPugJsOptions(&$input, &$filename, &$vars, &$pug)
             'out' => $workDirectory,
         );
         if (!empty($vars)) {
-            $options['obj'] = json_encode($vars, JSON_UNESCAPED_SLASHES);
+            $locals = json_encode($vars, JSON_UNESCAPED_SLASHES);
+
+            $options['obj'] = $locals;
+
+            if ($this->options['localsJsonFile']) {
+                $optionsFile = $workDirectory . '/options-' . hash('md5', time()) . '.json';
```

Comment:
Hi, I find the solution fine but `time()` should not be used for storage, imagine the following:
```php
$pug = new Pug(array(
    'pugjs' => true,
    'localsJsonFile' => true
);
$pug->render('foo.pug', $variablesForFoo);
$pug->render('bar.pug', $variablesForBar);
```
time() will probably be the same for foo and bar. I think here, tempnam would be appropriate: http://php.net/manual/en/function.tempnam.php